### DISCUSSION TITLE
show version of ZITADEL

### DISCUSSION THREAD

**Discussion by adlerhurst** [zitadel/zitadel#2695] [Ideas]
If I want to implement a client which calls ZITADEL, i would love to find the version of zitadel deployed. This would improve the maintainability of the client because I know with what version i have to build the client. So I would find out which fields are available.

I would implement it in the API and maybe also in the console so that admins see which version they use. Furthermore this would provide additional possibilities in case of support cases.

**Comment by livio-a** [comment]
There's an old issue for Console: 
- https://github.com/caos/zitadel/issues/151

The API should also support release channels
